interface/orders/single_order_results.inc.php

```diff
@@ -161,7 +160,7 @@ function generate_result_row(&$ctx, &$row, &$rrow, $priors_omitted = false)
         $ctx['priors_omitted'] = true;
     }
 
-// If a performing organization is provided, make a note for it also.
+    // If a performing organization is provided, make a note for it also.
     $result_facility = trim(str_replace("\r", "\n", $result_facility));
     if ($result_facility) {
```

Comment:
Change lines 177 to 179 to use the style background-color properties instead of bgcolor